test/spec/ModelerSpec.js

```diff
@@ -37,50 +37,66 @@ describe('Modeler', function() {
 
     setBpmnJS(modeler);
 
-    modeler.importXML(xml, function(err, warnings) {
-      done(err, warnings, modeler);
+    return new Promise(function(resolve) {
+      modeler.importXML(xml).then(function(result) {
+        resolve({ error: null, warnings: result.warnings, modeler: modeler });
+      }).catch(function(err) {
+        resolve({ error: err, warnings: err.warnings, modeler: modeler });
+      });
     });
   }
 
 
   it('should import simple process', function(done) {
     var xml = require('../fixtures/bpmn/simple.bpmn');
-    createModeler(xml, done);
+    createModeler(xml).then(function(result) {
+      done(result.error, result.warnings);
+    });
   });
 
 
   it('should import collaboration', function(done) {
     var xml = require('../fixtures/bpmn/collaboration-message-flows.bpmn');
-    createModeler(xml, done);
+    createModeler(xml).then(function(result) {
+      done(result.error, result.warnings);
+    });
   });
 
 
   it('should import nested lanes', function(done) {
     var xml = require('./features/modeling/lanes/lanes.bpmn');
-    createModeler(xml, done);
+    createModeler(xml).then(function(result) {
+      done(result.error, result.warnings);
+    });
   });
 
 
   it('should import ioSpecification', function(done) {
     var xml = require('./features/modeling/input-output/DataInputOutput.bpmn');
-    createModeler(xml, done);
+    createModeler(xml).then(function(result) {
+      done(result.error, result.warnings);
+    });
   });
 
 
   it.skip('should import complex', function(done) {
     var xml = require('../fixtures/bpmn/complex.bpmn');
-    createModeler(xml, done);
+    createModeler(xml).then(function(result) {
+      done(result.error, result.warnings);
+    });
   });
 
 
   it('should not import empty definitions', function(done) {
     var xml = require('../fixtures/bpmn/empty-definitions.bpmn');
 
     // given
-    createModeler(xml, function(err, warnings, modeler) {
+    createModeler(xml).then(function(result) {
```

Comment:
Can transform these tests into `awaitable` ones: Return a promise from a test and Mocha will wait for it to resolve or reject:

```javascript
    return createModeler(xml).then(function(result) {

      var modeler = result.modeler;

      // when
      return modeler.importXML(xml).catch(function(err) {

        // then
        expect(err.message).to.equal('no diagram to display');
      });
    });
```

Reply:
This way we get rid of `done` entirely where we can.